src/org/javarosa/core/model/utils/DateUtils.java

```diff
@@ -69,6 +68,26 @@ public DateFields () {
 //            tzOffset = 0;
         }
 
+        public DateFields(int year, int month, int day, int hour, int minute, int second, int secTicks) {
```

Comment:
Should this be private?

Reply:
You just made this constructor, didn’t you? And it’s only called from the same module. Anyway, no matter if you plan to remove it later.

Reply:
You're right! I didn't remember :/